web-src/src/helpers/FormatHelper.js

```diff
@@ -68,3 +68,16 @@ export function newGroupingLabelGenerator() {
     return null;
   };
 }
+
+export function getErrorCodeSubstring(str) {
```

Comment:
I'd recommend to change this method return format to be boolean and rename it to smth isLocalizationAvailable.